Kim/fade blink sketches

kimberleynyam opened the pull request.

kimberleynyam committed:
fade and blink sketches

Machinewarlock commented on kimnyamhondera/blink182/blink182.ino:
```diff
@@ -0,0 +1,18 @@
+/* This sketch is sketch for sketch this sketch task sketch https://trello.com/c/ba783WmU/2-blink-an-led */
```
Perhaps reword this comment? Too many 'sketch' being used. o3o

kimberleynyam replied:
I forgot to change this, I did this before i knew we'd be sharing it with the whole group

Machinewarlock commented on kimnyamhondera/blink182/blink182.ino:
```diff
@@ -0,0 +1,18 @@
+/* This sketch is sketch for sketch this sketch task sketch https://trello.com/c/ba783WmU/2-blink-an-led */
+
+int pin=13; /* declare pin */
```
You've specified the pin to be the value '13' so in the rest of your code where you have '13' you can just replace it with the word 'pin'

kimberleynyam replied:
didn't realise this until fade!

Machinewarlock commented on kimnyamhondera/fade/fade.ino:
```diff
@@ -0,0 +1,21 @@
+
```
Need to link to trello like you have done in the Blink code.

kimberleynyam replied:
need to get to grips with this

kimberleynyam committed:
sketches

Machinewarlock commented on kimnyamhondera/fade/fade.ino:
```diff
@@ -0,0 +1,21 @@
+int led = 6; /* declare pin */
+int brightness = 0; /* set initial brightness level */
+int fadeAmount = 5; /* set initial fadeAmount */
+
```
Require more comments elaborating on the code in your set up and loop.

mabatistussi commented on kimnyamhondera/blink182/blink182.ino:
```diff
@@ -0,0 +1,18 @@
+  digitalWrite(13, LOW); /*Light off */
+  delay(4000); /* 4 second delay */
+  
+}
```
I think your code is fine nothing else to add..

kimberleynyam replied:
thank you

mabatistussi commented on kimnyamhondera/fade/fade.ino:
```diff
@@ -0,0 +1,21 @@
+    }
+    
+  delay(50);
+}
```
I do believe you could add more comments to show up that you understand the code..that it!!!

kimberleynyam replied:
will take that on board. was just so pleased that this was working, everything else flew out of the window!

kimberleynyam committed:
sketches

kimberleynyam committed:
updated blink sketch

kimberleynyam committed:
updated fade

aliblackwell commented:
@kimberleynyam do you want to add the link to Trello in the comments? I'm relaxed either way. It's looking really good, well done.

kimberleynyam commented:
@aliblackwell yeah, someone else mentioned this. Do I just copy and paste the Trello checklist to my comments?

kimberleynyam committed:
multi-colour LED

Setup is set to blink each light once for one second at a time and loop
is set to fade each light.

kimberleynyam committed:
updated multiple LEDs

kimberleynyam committed:
updated multiple lights
